x-pack/plugin/runtime-fields/src/test/java/org/elasticsearch/xpack/runtimefields/query/StringScriptFieldRegexpQueryTests.java

```diff
@@ -59,7 +59,7 @@ protected StringScriptFieldRegexpQuery mutate(StringScriptFieldRegexpQuery orig)
                 pattern += "modified";
                 break;
             case 3:
-                flags = randomValueOtherThan(flags, () -> randomInt(0xFFFF));
+                flags = randomValueOtherThan(flags, () -> randomInt(RegExp.ALL));
```

Comment:
I think the change is ok but I wonder why that failed the test. We're supposed to rewrite the old all transparently. 
@markharwood can you check that the leniency is applied correctly ?

Reply:
There is a change that was not in the current form: https://github.com/apache/lucene-solr/pull/1659